Dockerfile

```diff
@@ -91,7 +91,7 @@ RUN pip install -r requirements-dev.txt
 RUN pip install tox==2.1.1
 
 ADD . /code/
-RUN python setup.py install
+RUN pip install --no-deps -e /code
```

Comment:
Oh nice - will this also prevent the problem we've had before where, if `setup.py` disagrees somehow with `requirements.txt`, then this step will silently reinstall different versions of dependencies?